### ISSUE TITLE
sqanti_qa report fails if there are no isoforms in a category

### ISSUE THREAD

**Issue by diekhans** [ConesaLab/SQANTI3#221]
If a category has no isoforms it, errors like the following occur:

 ```
        Quitting from lines 385-393 [p4.s1_p4.s2_p4.s3] (SQANTI3_report.Rmd)
          Error in `ans[ypos] <- rep(yes, length.out = len)[ypos]`:
          ! replacement has length zero
          Backtrace:
            1. ggplotly(p4.s3) %>% layout(showlegend = T, boxmode = "group")
            4. plotly:::ggplotly.ggplot(p4.s3)
            5. plotly::gg2list(...)
            6. plotly (local) ggplotly_build(p)
            7. layout$setup_panel_params()
               ...
           15. ggplot2:::expand_limits_scale(scale, expansion, limits, coord_limits = coord_limits)
           16. ggplot2:::expand_limits_discrete(...)
           17. ggplot2:::expand_limits_discrete_trans(...)
           18. ggplot2:::expand_limits_continuous_trans(...)
           19. base::ifelse(is.na(coord_limits), limits, coord_limits)
          Execution halted
```

This occurred by filtering all of the isoforms for novel gene categories and re-running sqanti on clean up set of isoforms.  I will be submitted a pull request that helped work around this.

**Comment by almart7** [comment]
Dear @diekhans  thank you so much for your input! We will review it as soon as possible.

**Comment by aarzalluz** [comment]
@diekhans merged PR #222, fix will be available in next SQANTI3 release. Thanks!

Issue #219 will remain open until we can further enhance the report plots to prevent errors when categories have no isoforms.
